docs/demo/app/views/examples/daisy/data_display/avatars.html.haml

```diff
@@ -0,0 +1,11 @@
+.prose.lg:prose-lg
+  %h2 Avatars
+
+  %p Here are some simple avatar examples.
+
+.flex.flex-row.items-center.gap-x-4
+  = daisy_avatar(css: "online", src: "https://img.daisyui.com/images/stock/photo-1534528741775-53994a69daeb.jpg") do
```

Comment:
I think we should come back later (in a different PR) and add pictures / initials for everyone who works on the project 😄